mslib/msui/qt5/ui_mscolab_connect_dialog.py

```diff
@@ -247,3 +263,13 @@ def retranslateUi(self, MSColabConnectDialog):
         self.idpAuthTokenSubmitBtn.setText(_translate("MSColabConnectDialog", "Submit"))
         self.idpAuthTopicLabel.setText(_translate("MSColabConnectDialog", "Identity Provider Authentication"))
         self.statusLabel.setText(_translate("MSColabConnectDialog", "Status:"))
+
+
```

Comment:
the `__name__` section  gets in by some conversion option, we don't need that

Reply:
okk